### DISCUSSION TITLE
What is the status of graphql subscriptions?

### DISCUSSION THREAD

**Discussion by dkamali-mc** [directus/directus#10428] [Archive]
Hey everyone, 

I really appreciate what you guys have been doing on here, I saw graphql subscriptions on the road map somewhere, if I remember correctly the target for that was the end of 2021, though I should ask as we are approaching year-end. 
 
 Thank you

**Comment by benhaynes** [comment]
We don't set official timelines unless a feature becomes sponsored. That said, we still aim/hope for `ws:` before EOY, but that may slip based on internal priorities or other sponsored work.
